Modules/TPC/src/CheckOfTrendings.cxx

```diff
@@ -0,0 +1,108 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   CheckOfTrendings.cxx
+/// \author Laura Serksnyte
+///
+
+#include "TPC/CheckOfTrendings.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+
+#include <fairlogger/Logger.h>
+#include <TCanvas.h>
+#include <TGraph.h>
+#include <TList.h>
+#include <TPaveText.h>
+
+#include <vector>
+#include <numeric>
+
+using namespace std;
```

Comment:
Please don't use this and `std::` where needed. You do it in most places anyhow.
```suggestion
```